### DISCUSSION TITLE
Enabling trace with configureStore?

### DISCUSSION THREAD

**Discussion by jessfeliciano** [reduxjs/redux-devtools#1273] [Q&A] [answered]
I love utilizing the chrome extension and decided I wanted to give setting up tracing  a go with a practice project. I noticed though there isn't any documentation that shows examples of implementing more advanced setting for the extension for `configureStore,` since `createStore` is deprecated. 

Is the current version not compatible yet with implementation through `configureStore`?

**Comment by markerikson** [comment, marked as answer]
We actually turn on the `trace` option by default in development builds inside of `configureStore`:

- https://github.com/reduxjs/redux-toolkit/blob/v1.9.0/packages/toolkit/src/configureStore.ts#L193-L198

But if you do need to alter the configuration, you can pass any Redux Devtools configuration options as the `devTools` option for `configureStore`:

- https://redux-toolkit.js.org/api/configureStore#devtools

We actually do that in my day job at Replay:

- [`src/ui/setup/store.ts`](https://github.com/replayio/devtools/blob/6a1b0839c20dd9b497893195674a9cadb7ed062f/src/ui/setup/store.ts#L75-L123)

**Reply by jessfeliciano** [reply]
This is really helpful, thank you!
